build.fsx

```diff
@@ -19,35 +27,58 @@ let sublimePath () =
     [  Environment.GetEnvironmentVariable("SUBLIME_TEXT_DATA")
        Environment.ExpandEnvironmentVariables(@"%APPDATA%\Sublime Text 3") ]
 
-  let isWindows = (Path.DirectorySeparatorChar = '\\')
   let searchPaths = if isWindows then WindowsPaths else UnixPaths
   let directories =
      searchPaths
      |> List.filter Directory.Exists
 
   match directories.Length with
-     | 0 ->
-         trace "No Sublime text 3 installation found"
+  | 0 -> trace "No Sublime text 3 installation found"
```

Comment:
Not a problem in this line or this commit specifically, I think, but I believe we're mixing tab lengths of 2 and 4 spaces?